### PR TITLE
#244 Add tslib to peerDependencies

### PR DESCRIPTION
**Pull request by scan-lan**
This is because the library is required in production

See issue #244

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -68,5 +69,5 @@
   "bugs": {
     "url": "https://github.com/MauricioRobayo/react-obfuscate-email/issues"
   },
-  "homepage": "https://www.mauriciorobayo/react-obfuscate-email"
+  "homepage": "https://www.mauriciorobayo.com/react-obfuscate-email"
```
Comment (MauricioRobayo) [inline package.json:72]: @scan-lan Could you please revert this line?

**Review by MauricioRobayo — CHANGES_REQUESTED** [review]
Awesome, thanks for the PR!

I'll be glad to merge this!

File: package.json (hunk 1 above)
Comment (scan-lan) [reply package.json]: Done :)

File: package.json [hunk 2]
```diff
@@ -20,7 +20,8 @@
     "build-docs": "build-storybook --docs --output-dir=docs"
   },
   "peerDependencies": {
-    "react": ">=16"
+    "react": ">=16",
+    "tslib": "^2.4.0"
```
[the remark this answers is not in the document]
Comment (scan-lan) [reply package.json:24]: Yeah, Mauricio, I agree regarding >= 2.

I'll update the PR once I'm home :)

**Comment by scan-lan** [comment]
Hi, Mauricio.

Rebased and updated tslib to `>=2`. That should be it :)

**Review by MauricioRobayo — APPROVED** [review]
Thanks, Luke!

**Comment by scan-lan** [comment]
My pleasure!

**Pull request merged**